src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -97,15 +120,16 @@ def _parse_fieldop_arg(
     if not isinstance(arg, Field):
         raise ValueError(f"Received {node} as argument to field operator, expected a field.")
 
+    def make_iterator_index(var: str) -> gtir_dataflow.SymbolExpr:
+        dtype = dace.dtype_to_typeclass(gtx_fbuiltins.IndexType)
+        return gtir_dataflow.SymbolExpr(var, dtype)
+
```

Comment:
This is a private function, it can not be used everywhere else, however, you are only using it once in this function.
This looks a bit strange.